ansible/jenkins.yml

```
@@ -0,0 +1,135 @@
+  - name: Install kind (Kubernetes in Docker)
+    when: ansible_architecture == "x86_64"
+    block:
+      - name: Download kind binary
+        get_url:
+          url: https://kind.sigs.k8s.io/dl/v0.27.0/kind-linux-amd64
+          dest: /usr/local/bin/kind
+          mode: '0755'
+
+  - name: Install kubectl
+    block:
+      - name: Set kubectl version
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)